text/0000-build-write-flag.md

```diff
@@ -0,0 +1,121 @@
+# Meta
+[meta]: #meta
+- Name: Write flag for build layers
+- Start Date: 2021-04-15
+- Author(s): [@samj1912](https://github.com/samj1912)
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+This RFC proposes the introduction of a  `build-write` flag in addition to the `build` flag. The idea behind this flag is to expose build layers that are both read-write (if `build-write` is set to `true`) or build layers that are read-only (if `build-write` is set to `false`) for subsequent buildpacks.
+
+# Definitions
+[definitions]: #definitions
+
+Build layers: Layers from a specific buildpack that are available to subsequent buildpacks during the build process.
+layerize: The process of converting a layer directory to a tarball which will be used to create the final application image during the export phase.
```

Comment:
```suggestion
Build layers: Layers from a specific buildpack that are available to subsequent buildpacks during the build process.

layerize: The process of converting a layer directory to a tarball which will be used to create the final application image during the export phase.
```

You can either add an empty line between the definitions or just add them in bullets. Just so they won't appear in one long line.